### PR TITLE
Podman V2 birth

### PR DESCRIPTION
**Pull request by baude**
remote podman v1 and replace with podman v2.

Signed-off-by: Brent Baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
Yikes, Lets get this in once you can pass limited tests.

File: cmd/podman/README.md [hunk 1]
```diff
@@ -1,15 +1,113 @@
...
+    manifestCmd.SetUsageTemplate(registry.UsageTemplate())
+}
+
+// preRunE populates the image engine for sub commands
+func preRunE(cmd *cobra.Command, args []string) error {
+    _, err := registry.NewImageEngine(cmd, args)
+    return err
+}
+```
+To "wire" in the `manifest` command, edit the file ```$GOPATH/src/github.com/containers/libpod/cmd/podmanV2/main.go``` to add:
```
Comment (haircommander) [inline cmd/podman/README.md:65]: nit, wouldn't this be 
`Create the file ```$GOPATH/src/github.com/containers/libpod/cmd/podman/manifests/inspect.go`
now?

Comment (baude) [reply cmd/podman/README.md:65]: yeah

Comment (baude) [reply cmd/podman/README.md:65]: we take prs

Comment (haircommander) [reply cmd/podman/README.md:65]: but you're so good at them :smiley:

**Comment by jwhonce** [comment]
LGTM

File: .cirrus.yml [hunk 2]
```diff
@@ -329,10 +337,11 @@ meta_task:
...
             ${FEDORA_CACHE_IMAGE_NAME}
-            ${PRIOR_FEDORA_CACHE_IMAGE_NAME}
-            ${UBUNTU_CACHE_IMAGE_NAME}
-            ${PRIOR_UBUNTU_CACHE_IMAGE_NAME}
-            ${IMAGE_BUILDER_CACHE_IMAGE_NAME}
+            # FIXME
+            #${PRIOR_FEDORA_CACHE_IMAGE_NAME}
+            #${UBUNTU_CACHE_IMAGE_NAME}
+            #${PRIOR_UBUNTU_CACHE_IMAGE_NAME}
+            #${IMAGE_BUILDER_CACHE_IMAGE_NAME}
```
Comment (cevich) [inline .cirrus.yml:344]: I think this is a mistake.  My understanding is that we want these VM images to NOT get pruned.  That means they need their timestamps updated by `meta_task`.  That won't happen if `meta_task` doesn't run or runs without any image names :smile:

File: .cirrus.yml [hunk 3]
```diff
@@ -313,6 +320,7 @@ build_without_cgo_task:
 # Update metadata on VM images referenced by this repository state
 meta_task:
 
+    skip: $CI == 'true'
```
Comment (cevich) [inline .cirrus.yml:323]: We probably want `meta_task` to run.

**Comment by cevich** [comment]
LGTM

**Pull request merged**